src/Theme.cpp

```diff
@@ -1281,10 +1292,15 @@ auStaticText::auStaticText(wxWindow* parent, wxString textIn) :
    SetForegroundColour( theTheme.Colour( clrTrackPanelText));
    SetName(textIn);
    SetLabel(textIn);
-}
- 
-void auStaticText::OnPaint(wxPaintEvent & WXUNUSED(evt))
-{
+    }
+    <<<<<<< HEAD
+    
+    void auStaticText::OnPaint(wxPaintEvent & WXUNUSED(evt))
+    =======
+    
+    void auStaticText::OnPaint(wxPaintEvent & evt)
+    >>>>>>> 5d38ecf9a... Added Pro Tools theme
```

Comment:
Can you explain those "comments"? They don't look like comments to me and they look like they will make errors.

Reply:
Thanks for pointing that out. And you're right. Those comments is just part of something I've been working on. Forgot to remove it. I'll push an update.